Detectors/TRD/workflow/include/TRDWorkflow/TRDEventDisplayFeedSpec.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include <gsl/span>
+
+#include "TRDBase/Geometry.h"
+
+#include "Framework/DataProcessorSpec.h"
+#include "Framework/Task.h"
+
+#include "DataFormatsGlobalTracking/RecoContainer.h"
+#include "DataFormatsTRD/TrackTRD.h"
+#include "DataFormatsTRD/TriggerRecord.h"
+#include "DataFormatsTRD/Digit.h"
+
+#include "TRDBase/TrackletTransformer.h"
+
+namespace o2
+{
+namespace trd
+{
+
+class TRDEventDisplayFeedSpec : public o2::framework::Task
+{
+ public:
+  TRDEventDisplayFeedSpec(std::shared_ptr<o2::globaltracking::DataRequest> dataRequest) : mDataRequest(dataRequest){};
+  ~TRDEventDisplayFeedSpec() override = default;
```

Comment:
Should I close it at the end of `TRDEventDisplayFeedSpec::run()`?

Reply:
Probably, since you open it at the beginning of that function. And I would also change it from `std::ofstream* summary` to `std::ofstream mSummary` (naming convention + avoid usage of raw `new` which is also not deleted anywhere)